docs/solana/rewardable_entities.mdx

```diff
@@ -403,6 +409,5 @@ const tx = await formTransaction({
   lazyDistributor: lazyDistributorPkey,
 })
 
-await provider.send(tx);
+await provider.send(tx)
```

Comment:
Prettier again 🤔

Reply:
think ive got a fix for these, adding `<!-- prettier-ignore -->` before a code block makes prettier ignore the code block

example:

![carbon](https://user-images.githubusercontent.com/1404219/215610420-a30e6b37-aefc-4d9b-a251-a5a2f4f2a74c.png)